code/game/gamemodes/colonialmarines/whiskey_outpost.dm

```diff
@@ -322,9 +322,9 @@
 		OT = "sup" //no breaking anything.
 
 	else if (OT == "sup")
-		randpick = rand(0,50)
+		randpick = rand(0,84)
```

Comment:
There are numbers up to 90

Reply:
i forgot to adjust when changing the weightings... woops
```suggestion
		randpick = rand(0,90)
```